internal/standalone/standalone.go

```diff
@@ -15,11 +16,14 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/grafana/grafana-plugin-sdk-go/build"
 	"github.com/grafana/grafana-plugin-sdk-go/internal"
 )
 
 var (
 	standaloneEnabled = flag.Bool("standalone", false, "should this run standalone")
+	buildInfoMode     = flag.Bool("buildinfo", false, "print build info and exit")
```

Comment:
Nit: Feels a bit weird to put these in the `standalone` package - curious on the reasoning there in case I'm missing something 🤔